dispatchers.go

```diff
@@ -187,6 +187,9 @@ func from(b *Builder, args []string, attributes map[string]bool, flagArgs []stri
 			return fmt.Errorf("Windows does not support FROM scratch")
 		}
 	}
+	if len(args) ==3 {
```

Comment:
Please always use gofmt in your editor and format on save.  We don't have the bot on for this repo yet, but it would be required in almost any of them.